frontend/src/scenes/session-recordings/player/snapshot-processing/process-all-snapshots.ts

```diff
@@ -76,16 +75,6 @@ export function processAllSnapshots(
         const sourceResult: RecordingSnapshot[] = []
 
         for (const snapshot of sourceSnapshots) {
```

Comment:
this was added in reaction to a bug for sure
(i forget what exactly since it was a while ago)

but so I guess the real thing here is if we can very cheaply detect duplicates
and only do the deduplication when necessary